resources/views/components/frameworks/bootstrap5/table-base.blade.php

```diff
@@ -5,15 +5,14 @@
                     'enabledFilters' => $enabledFilters
             ])
 
-            @includeIf($theme->layout->message)
-        </div>
-        <div class="table-responsive col-md-12" style="margin: 10px 0 10px;">
-            @include($table)
-        </div>
-        <div class="row">
-            <div class="col-12 overflow-auto">
-                @include($theme->footer->view, ['theme' => $theme])
-            </div>
+        @includeIf($theme->layout->message)
+    </div>
+    <div class="table-responsive overflow-visible col-md-12" style="margin: 10px 0 10px;">
```

Comment:
removed overflow-visible because it broke bootstrap layout on mobile

Reply:
Could you suggest any alternatives for this?
The problem is that if we don't have rows in the table (or there are very few of them), the height of the table is too small to display a long list of the select/multiselect/datepicker filter well